.github/workflows/master-push.yml

```diff
@@ -38,7 +39,6 @@ jobs:
         with:
           name: k-framework
           authToken: '${{ secrets.CACHIX_PUBLIC_TOKEN }}'
-          skipPush: true
```

Comment:
I think we need @goodlyrottenapple to have a look. I though that the actual pushing to cache was done by the `cachix push k-framework` at the end of the 'Build and cache kavm' step.

Reply:
Reverted.